src/core/server/elasticsearch/client/configure_client.test.ts

```diff
@@ -384,7 +385,7 @@ describe('configureClient', () => {
           Array [
             Array [
               "400
-          GET /_path [undefined]: Response Error",
+          GET /_path [undefined]: {\\"error\\":{}}",
```

Comment:
The client changed the serialization mechanism for errors that don't look like Elasticsearch errors https://github.com/elastic/elasticsearch-js/pull/1509